<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarProfilesWidget</class>
 <widget class="QWidget" name="ToolBarProfilesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profiles</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Main">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_Content">
     <item>
      <widget class="QGroupBox" name="groupBox_Profiles">
       <property name="title">
        <string>Profile Management</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_Profiles">
        <property name="topMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget_Profiles">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_ProfileActions">
          <item>
           <widget class="QPushButton" name="pushButton_SaveProfile">
            <property name="text">
             <string>Save Current as New Profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_LoadProfile">
            <property name="text">
             <string>Load Profile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_DeleteProfile">
            <property name="text">
             <string>Delete Profile</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_ProfileDetails">
       <property name="title">
        <string>Profile Details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_ProfileDetails">
        <property name="topMargin">
         <number>30</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_ProfileName">
          <property name="text">
           <string>Profile Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ProfileName">
          <property name="placeholderText">
           <string>Enter profile name...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_ProfileDescription">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_ProfileDescription">
          <property name="placeholderText">
           <string>Optional description...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_ProfileContents">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Profile includes:&lt;/span&gt; Power profile mode, CPU governor and frequency settings, SMT configuration, fan curve points, CPU voltage offsets, GPU clock offsets, Other settings except GPU mode.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::RichText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
